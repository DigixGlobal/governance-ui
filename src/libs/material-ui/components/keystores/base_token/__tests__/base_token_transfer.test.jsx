import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import BaseTokenTransfer from '../base_token_transfer';

describe('<BaseTokenTransfer />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <BaseTokenTransfer
        trigger={<div />}
        network={{
          name: 'test network',
        }}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
