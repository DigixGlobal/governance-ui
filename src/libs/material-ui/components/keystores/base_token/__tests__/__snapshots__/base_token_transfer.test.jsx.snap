// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseTokenTransfer /> renders correctly 1`] = `
<TransactionModal
  data={Object {}}
  form={[Function]}
  handleTransaction={[Function]}
  header="Send test network Ether"
  network={
    Object {
      "name": "test network",
    }
  }
  onMined={[Function]}
  trigger={<div />}
/>
`;
